guides/source/action_controller_overview.md

```diff
@@ -710,7 +710,7 @@ In addition to "before" filters, you can also run filters after an action has be
 
 "around" filters are responsible for running their associated actions by yielding, similar to how Rack middlewares work.
 
-For example, in a website where changes have an approval workflow an administrator could be able to preview them easily, just apply them within a transaction:
+For example, in a website where changes have an approval workflow, an administrator could preview them easily by applying them within a transaction:
```

Comment:
I think the `them` in this sentence could read better too. Not sure what the best change would be, though. My first thought was changing the "them" to "changes", but that wasn't any better.

Reply:
I see what you mean. It's a bit ambiguous whether "them" refers to changes or the approval workflows.